libs/service-portal/core/src/lib/navigation/masterNavigation.ts

```diff
@@ -103,8 +130,23 @@ export const servicePortalMasterNavigation: ServicePortalNavigationItem[] = [
         children: [
           {
             navHide: true,
-            name: m.detailInfo,
-            path: ServicePortalPath.LicensesDetail,
+            name: m.drivingLicense,
+            path: ServicePortalPath.DrivingLicensesDetail,
+          },
+          {
+            navHide: true,
+            name: m.firearmLicense,
+            path: ServicePortalPath.FirearmLicensesDetail,
+          },
+          {
+            navHide: true,
+            name: m.adrLicense,
+            path: ServicePortalPath.ADRLicensesDetail,
+          },
+          {
+            navHide: true,
+            name: m.machineLicense,
+            path: ServicePortalPath.MachineLicensesDetail,
```

Comment:
Disability license is missing

Reply:
Good catch, but ... for this case, that should be fine.